test/conformance/source/environment.cpp

```diff
@@ -43,8 +43,13 @@ std::ostream &operator<<(std::ostream &out,
 uur::PlatformEnvironment::PlatformEnvironment(int argc, char **argv)
     : platform_options{parsePlatformOptions(argc, argv)} {
     instance = this;
+
+    ur_loader_config_handle_t config;
+    urLoaderConfigCreate(&config);
+    urLoaderConfigEnableLayer(config, "UR_LAYER_FULL_VALIDATION");
```

Comment:
Should be bail out if these calls results in an error?

Reply:
yeah all should have been checked, the way we handle errors in this area isn't ideal for this many API calls happening.. I've made an attempt to have checks for everything while making sure stuff gets freed if we need to bail